state/pgstatestorage/l1infotree.go

```diff
@@ -0,0 +1,48 @@
+package pgstatestorage
+
+import (
+	"context"
+
+	"github.com/0xPolygonHermez/zkevm-node/state"
+	"github.com/jackc/pgx/v4"
+)
+
+// AddL1InfoRootToExitRoot adds a new entry in ExitRoot and returns index of L1InfoTree and error
+func (p *PostgresStorage) AddL1InfoRootToExitRoot(ctx context.Context, exitRoot *state.L1InfoTreeExitRootStorageEntry, dbTx pgx.Tx) (uint64, error) {
+	const addGlobalExitRootSQL = `INSERT INTO state.exit_root
+				(block_num, timestamp, mainnet_exit_root, rollup_exit_root, global_exit_root, prev_block_hash, l1_info_root, l1_info_tree_index)
+		VALUES ($1,         $2,        $3,                $4,               $5,               $6,              $7,
+			COALESCE((SELECT MAX(l1_info_tree_index) + 1 FROM state.exit_root), 1))
```

Comment:
index must start with 0

Reply:
@ARR552 I think that 0 is a special value for the L1infotree that means 'no change', so the first leaf is 1